nocodes/src/main/java/io/qonversion/nocodes/internal/di/network/NetworkAssemblyImpl.kt

```diff
@@ -53,4 +55,13 @@ internal class NetworkAssemblyImpl(
             retryPolicy
         )
     }
+
+    private fun isFallbackActuallyAvailable(): Boolean {
+        return try {
+            val effectiveFileName = internalConfig.primaryConfig.effectiveFallbackFileName
+            context.assets.open(effectiveFileName).use { true }
+        } catch (e: Exception) {
+            false
+        }
+    }
```

Comment:
I think, it's not assembly responsibility. Let's provide this information from outside?